generators/entity-server/files.js

```diff
@@ -105,19 +100,32 @@ const serverFiles = {
             ],
         },
         {
-            condition: generator =>
-                generator.reactive &&
-                ['mongodb', 'cassandra', 'couchbase', 'neo4j'].includes(generator.databaseType) &&
-                !generator.embedded,
+            condition: generator => generator.reactive && !generator.embedded,
             path: SERVER_MAIN_SRC_KOTLIN_DIR,
             templates: [
                 {
-                    file: 'package/repository/EntityReactiveRepository.kt',
+                    file: 'package/repository/EntityRepository_reactive.kt',
```

Comment:
I think the class' name should be camel cased

Reply:
Yeah it will be renamed to camel case. Can you check the next line?

Reply:
ohhh, you sneaky! Sorry for missing that